module/contrib/local/protocgen.sh

```diff
@@ -9,15 +9,9 @@ for dir in $proto_dirs; do
   -I "third_party/proto" \
   --gocosmos_out=plugins=interfacetype+grpc,\
 Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
+  --grpc-gateway_out=logtostderr=true:. \
   $(find "${dir}" -maxdepth 1 -name '*.proto')
-
-  # # command to generate gRPC gateway (*.pb.gw.go in respective modules) files
-  # buf protoc \
```

Comment:
Does this work?

Reply:
yea, you can add as many generators to a single protoc command as you want